source/simulator/parameters.cc

```diff
@@ -963,25 +963,26 @@ namespace aspect
                          "violating this condition is that the pressure may show "
                          "oscillations and not converge to the correct pressure."
                          "\n\n"
-                         "That said, people have often used $Q_1$ elements for both the"
+                         "That said, people have often used $Q_1$ elements for both the "
                          "velocity and pressure anyway. This is commonly referred to as "
-                         "using the Q1-Q1 method. It is, by default, not stable as "
+                         "using the $Q_1-Q_1$ method. It is, by default, not stable as "
                          "mentioned above, but it can be made stable by adding small "
                          "amount of compressibility to the model. There are numerous "
                          "ways to do that. Today, the way that is generally considered "
                          "to be the best approach is the one by Dohrmann and Bochev "
                          "\\cite{DohrmannBochev2004}."
                          "\n\n"
                          "When this parameter is set to ``true'', then \\aspect{} "
-                         "will use this method by using $Q_k\times Q_k$ elements for "
+                         "will use this method by using $Q_k\\times Q_k$ elements for "
                          "velocity and pressure, respectively, where $k$ is the value "
                          "provided for the parameter ``Stokes velocity polynomial "
                          "degree''."
                          "\n\n"
                          "\\note{While \\aspect{} \\textit{allows} you to use this "
                          "  method, it is generally understood that this is not a "
                          "  great idea as it leads to rather low accuracy in "
-                         "  general. It also leads to substantial problems when "
+                         "  general as documented in Thieulot and Bangerth \\cite{thba22}. "
```

Comment:
Does this way of citing repeat the author names? Or does \cite{} only give the year?

Reply:
Looking at all of the other places in the `.cc` files where we `\cite` something, the large majority seems to not duplicate names and just have the reference. So I would suggest
```suggestion
                         "  general as documented in \\cite{thba22}. "
```